lms/static/sass/features/_course-experience.scss

```diff
@@ -1,11 +1,33 @@
 // ------------------------------
 // Styling for files located in the openedx/features repository.
 
+// Course sidebar sections
+.course-sidebar {
+  .section {
+    h3 {
```

Comment:
Styles should be applied to classes instead of elements. Especially important for headings given their a11y significance.

Reply:
The challenge here is that arbitrary HTML is rendered by course teams into the course dates section. I'll try adding a class and see if it still works okay.

Reply:
I remember the nightmare A.J. had adjusting headings for an a11y review, which really hit home the importance of not styling elements directly.

Reply:
I've made this change and it seems fine.

Reply:
Great, if you can commit that change to this PR that would be good.

Reply:
Done